### PR TITLE
Spanish translations improvement

### PR DESCRIPTION
**Pull request by bombozama**
* Synced array order to improve readability and maintenance.
* Removed non existing entries
* Added and translated missing lines

### CODE REVIEW

File: lang/es/lang.php [hunk 1]
```diff
@@ -8,31 +8,45 @@
...
         'activate_warning_title' => '¡El usuario no se pudo activar!',
         'activate_warning_desc' => 'Este usuario todavía no ha sido activado y no será capaz de iniciar sesión.',
         'activate_confirm' => '¿Realmente desea activar este usuario?',
         'activated_success' => '¡El usuario ha sido activado exitosamente!',
         'activate_manually' => 'Activar usuario manualmente',
+        'convert_guest_confirm' => '¿Convertir este invitado en usuario?',
+        'convert_guest_manually' => 'Convertir a usuario registrado',
+        'convert_guest_success' => 'El usuario ha sido convertido a una cuenta registrada',
+        'impersonate_user' => 'Iniciar sesión como este usuario',
+        'impersonate_confirm' => '¿Estás seguro que quieres iniciar sesión como este usuario? Puede volver a tu estado original cerrando la sesión.',
```
Comment (noiarek) [inline lang/es/lang.php]: Dice `Estás` y luego `Puede`. Creo que lo correcto seria Puedes ya que al inicio empieza informal, que opinas?

File: lang/es/lang.php [hunk 2]
```diff
@@ -8,31 +8,45 @@
         'access_users' => 'Administrar usuarios',
         'access_groups' => 'Administrar grupos',
         'access_settings' => 'Preferencias de usuario',
+        'impersonate_user' => 'Personificar Usuarios'
```
Comment (noiarek) [inline lang/es/lang.php]: Creo que lo correcto seria Usuario, que opinas?

File: lang/es/lang.php (hunk 1 above)
Comment (bennothommo) [reply lang/es/lang.php]: @noiarek Thanks for your comments. Just so we can track the changes being requested, would you mind translating your request to @bombazama in English as well?

Comment (bombozama) [reply lang/es/lang.php]: Fixed in f9523328ddcded2dc8b7ce675a7b163d6687bbaa
Thanks!

File: lang/es/lang.php (hunk 2 above)
Comment (bombozama) [reply lang/es/lang.php]: Fixed in f9523328ddcded2dc8b7ce675a7b163d6687bbaa
Thanks!

File: lang/es/lang.php [hunk 3]
```diff
@@ -8,31 +8,45 @@
...
         'access_groups' => 'Administrar grupos',
         'access_settings' => 'Preferencias de usuario',
+        'impersonate_user' => 'Personificar Usuario'
     ],
     'users' => [
         'menu_label' => 'Usuarios',
         'all_users' => 'Todos los usuarios',
         'new_user' => 'Nuevo usuario',
         'list_title' => 'Administrar usuarios',
-        'activating' => 'Activando...',
```
Comment (LukeTowers) [inline lang/es/lang.php:17]: Why was this key removed?

Comment (bombozama) [reply lang/es/lang.php:17]: I couldn't find it in the english file (ran a diff). I did some digging and it was removed from EN in 7b47166bc7a4f47dba139d54f9d0d9622fc3ce41

Comment (LukeTowers) [reply lang/es/lang.php:17]: Gotcha, thanks for checking!

**Pull request merged**